manifests/params.pp

```diff
@@ -9,19 +9,8 @@
     'RedHat': {
       case $facts['os']['name'] {
         'AlmaLinux', 'Rocky', 'RedHat', 'CentOS', 'OracleLinux', 'Scientific', 'OEL', 'SLC', 'CloudLinux': {
-          if (versioncmp($facts['os']['release']['full'], '5.0') < 0) {
-            $jdk_package = 'java-1.6.0-sun-devel'
-            $jre_package = 'java-1.6.0-sun'
-            $java_home   = '/usr/lib/jvm/java-1.6.0-sun/jre/'
-          }
-          # See cde7046 for why >= 5.0 < 6.3
-          elsif (versioncmp($facts['os']['release']['full'], '6.3') < 0) {
-            $jdk_package = 'java-1.6.0-openjdk-devel'
-            $jre_package = 'java-1.6.0-openjdk'
-            $java_home   = '/usr/lib/jvm/java-1.6.0/'
-          }
           # See PR#160 / c8e46b5 for why >= 6.3 < 7.1
-          elsif (versioncmp($facts['os']['release']['full'], '7.1') < 0) {
```

Comment:
we are good to remove reference of Debian 9 :
```
'9': {
          $java = {
            'jdk' => {
              'package'          => 'openjdk-8-jdk',
              'alternative'      => "java-1.8.0-openjdk-${openjdk_architecture}",
              'alternative_path' => "/usr/lib/jvm/java-1.8.0-openjdk-${openjdk_architecture}/bin/java",
              'java_home'        => "/usr/lib/jvm/java-1.8.0-openjdk-${openjdk_architecture}/",
            },
            'jre' => {
              'package'          => 'openjdk-8-jre-headless',
              'alternative'      => "java-1.8.0-openjdk-${openjdk_architecture}",
              'alternative_path' => "/usr/lib/jvm/java-1.8.0-openjdk-${openjdk_architecture}/bin/java",
              'java_home'        => "/usr/lib/jvm/java-1.8.0-openjdk-${openjdk_architecture}/",
            },
          }
        }
```

Reply:
Removed.